ui/packages/catalog-ui-search/src/main/webapp/component/selection-checkbox/selection-checkbox.view.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+const Marionette = require('marionette')
+import React from 'react'
+import styled from '../../react-component/styles/styled-components'
+
+const Root = styled.div`
+  display: block;
+  width: ${props => props.theme.minimumSpacing};
+  padding-right: ${props => props.theme.minimumButtonSize};
+  cursor: pointer;
+`
+
+module.exports = Marionette.ItemView.extend({
+  events: {
+    'click span': 'handleClick',
+  },
+  handleClick(e) {
+    e.stopPropagation()
```

Comment:
In the end, this shouldn't be necessary.  We can go without passing an onClick into the checkboxes that are within the rows and the result item since they have an outer onClick that will handle things for us.